### PR TITLE
Kgp 396 central email notification

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
image

and update menu
image

### CODE REVIEW

File: src/pages/Verify/index.tsx [hunk 1]
```diff
@@ -70,12 +65,8 @@
       return
     }
     refTimeoutCountDown.current && clearInterval(refTimeoutCountDown.current)
-    if (isVerifyExternal) {
-      if (redirectUrl) window.location.href = redirectUrl
-    } else {
-      navigate(APP_PATHS.SWAP)
-    }
-  }, [isVerifyExternal, navigate, redirectUrl])
+    if (redirectUrl) window.location.href = redirectUrl
```
[the remark this answers is not in the document]
Comment (nguyenhoaidanh) [reply src/pages/Verify/index.tsx:68]: checked in `validateUrl` function

[the remark this answers is not in the document]
Comment (nguyenhoaidanh) [reply src/pages/Verify/index.tsx:68]: checked in `validateUrl` function

File: src/components/Menu/index.tsx [hunk 2]
```diff
@@ -11,48 +11,61 @@ import {
...
+import { isChristmasTime } from 'utils'
 
 import ClaimRewardModal from './ClaimRewardModal'
 import FaucetModal from './FaucetModal'
 import NavDropDown from './NavDropDown'
 
 const MenuItem = styled.li`
   flex: 1;
-  padding: 0.75rem 0;
+  padding: 0.7rem 0;
```
Comment (namgold) [inline src/components/Menu/index.tsx:61]: evil number

Comment (nguyenhoaidanh) [reply src/components/Menu/index.tsx:61]: tell me why

Comment (namgold) [reply src/components/Menu/index.tsx:61]: 0.7 * 16 = 11.2 px lol. Why would we need that little decimal px

Comment (nguyenhoaidanh) [reply src/components/Menu/index.tsx]: image
lol, but I reverted.

Comment (namgold) [reply src/components/Menu/index.tsx]: image

He missed `1`  maybe? :joy:

**Review by namgold — APPROVED** [review]

**Review by namgold — APPROVED** [review]

**Pull request merged**